src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -225,6 +225,11 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'default': true,
 			'description': localize('notebook.enableDoubleClickEdit', "Enable double click to edit for text cells in notebooks")
+		},
+		'notebook.setWYSIWYG ByDefault': {
```

Comment:
The space is by design here, due to how the settings titles are populated. Otherwise, users would see something like:

Set WYSIWYGBy Default